script.js

```diff
@@ -1,11 +1,22 @@
-// Create an array called `colors` that contains three colors.
+var colors = ['green', 'yellow', 'red'];
 
 // Log the last item in the array to the console.
 
-// Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
+   console.log(colors[2]);
 
+// Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
+for(i = 0; i < colors.length; i++){
+  var text = "";
+  text += "My favoriate color is" + " " + colors[i];
```

Comment:
👍 Your `for` loop is good and references the `colors` array perfectly.

When we ask for something to be printed though, typically we're referring to the console output so you'll want to use `console.log()`.

```js
console.log("My favoriate color is" + " " + colors[i]);
```